### PR TITLE
Fix #26.

### LINKED ISSUE #26: Add to album dialog not working
**How to reproduce:**
1. Select one or more picture on photos page
2. Click plus (+) sign

**Expected:** Dialog to select an album appears

**Actual:** Background is darkened, but no dialog shows up.

### PR DESCRIPTION
**Pull request by muety**

### CODE REVIEW

**Review by spaibs — APPROVED** [review]

File: lib/main.dart [hunk 1]
```diff
@@ -166,46 +150,49 @@ class MainPage extends StatelessWidget {
...
-                        addPhotosToAlbum(
-                            Albums.getAlbumList(context)[index].id, context);
-                      },
-                      child: Card(
-                          child: ListTile(
-                              title: Text(
-                                  Albums.getAlbumList(context)[index].name))));
-                }),
+            content: Container(
+              width: double.maxFinite,
```
Comment (thielepaul) [inline lib/main.dart:154]: @n1try as I don't have much experience in Flutter, I have one small question regarding the change: what is the effect of wrapping the ListView with a Container and setting the width to `double.maxFinite`?

**Comment by thielepaul** [comment]
@n1try Thank you for the pull request!
I was just wondering which code formatter are you using? If I run `flutter format lib/main.dart`, a lot of lines are changed, which is suboptimal as it gets harder to track changes and unnecessary merge conflicts might occur.

File: lib/main.dart (hunk 1 above)
Comment (muety) [reply lib/main.dart:154]: To be honest I essentially searched for the error message and found [this fix](https://stackoverflow.com/a/56355962/3112139). Without being too proficient in Flutter, my bare understanding is that when not wrapping the list inside a container, the renderer can't know the lists horizontal and vertical dimensions before rendering each and every item, which is costly. The container provides kind of a constrained frame around the potentially endlessly large list. I can't particularly explain why setting the width to effectively infinite is helpful. I would assume that setting it to the actual screen width would have the same effect.

**Comment by muety** [comment]
> I was just wondering which code formatter are you using?

I tried using Android Studio's default formatting, but that changed everything entirely. Instead I manually changed some indentations that I found overly ugly (e.g. [here](https://github.com/photoprism/photoprism-mobile/pull/27/files#diff-fe53fad46868a294b309fc85ed138997L88)), but I agree that merging gets harder by doing so. Feel free to revert my formatting changes or maybe even add some kind of formatting specification (like [editorconfig](https://editorconfig.org/) or so?) to have consistency while avoiding ugly indentations.

File: lib/main.dart (hunk 1 above)
Comment (thielepaul) [reply lib/main.dart:154]: ok, this seems to make sense, it's interesting that the problem only occurs on some devices.
Thanks, for fixing it :+1:

Comment (muety) [reply lib/main.dart:154]: It's only my understanding, no guarantee for correctness :wink: 
According to the post I shared above, it's exactly all phones with Android < 9.0 that cause the problem.

**Comment by thielepaul** [comment]
For the start, let's just do what the Flutter framework suggests: https://flutter.dev/docs/development/tools/formatting
To enforce the formatting, I also added an check to the pipeline/Github actions (#28).
Anyway, I am open for other suggestions regarding the formatting :)

**Pull request merged**